### PR TITLE
feat: set up unit tests

### PR DESCRIPTION
**Pull request by Dripcoding**
Set up unit tests using jest and react-testing-library

- create test directories
- create simple unit test to confirm unit test set up works

### CODE REVIEW

File: facade-app/src/test/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react'
+
+import Footer from '../Components/Footer';
+
+describe('Footer', () => {
+    it('renders correctly', () => {
+        const { container } = render(<Footer />);
+
+        expect(container).toMatchSnapshot();
```
Comment (Dripcoding) [inline facade-app/src/test/Footer.test.js:10]: wrote this snapshot test to confirm tests configured correctly

File: facade-app/src/test/__snapshots__/Footer.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      class="rs-navbar rs-navbar-default"
+      role="navigation"
+    >
+      <div
+        class="rs-navbar-body"
+        style="text-align: center; height: 30px; font-size: 15px;"
+      >
+        <div>
+          © Copyright 
+          2022
```
Comment (preetkaran20) [inline facade-app/src/test/__snapshots__/Footer.test.js.snap:18]: this will fail next year as date will change.

Comment (preetkaran20) [reply facade-app/src/test/__snapshots__/Footer.test.js.snap:18]: make this as dynamic.

**Review by preetkaran20 — COMMENTED** [review]
A small fix, over all, LGTM.

File: facade-app/src/test/__snapshots__/Footer.test.js.snap (hunk 2 above)
Comment (Dripcoding) [reply facade-app/src/test/__snapshots__/Footer.test.js.snap:18]: good point, changed this so the year can be passed as a prop

**Comment by Dripcoding** [comment]
made the suggested change. Could you enable merge permissions for collaborators?

**Review by preetkaran20 — APPROVED** [review]

**Comment by preetkaran20** [comment]
LGTM thanks a lot

**Comment by preetkaran20** [comment]
@Dripcoding there is a problem with the PR and it caused build failure in github action. Can you go to github actions and fix that?

**Comment by preetkaran20** [comment]
https://github.com/SasanLabs/VulnerableApp-facade/actions/runs/2013536127

**Pull request merged**